lib/api/2.0/nodes.js

```diff
@@ -78,9 +77,7 @@ var nodesGetWorkflowById = controller(function(req) {
         newQuery = req.query;
     }
 
-    return workflowApiService.getWorkflowsByNodeId(req.swagger.params.identifier.value,
-                                                   newQuery);
-
+    return nodes.getNodeWorkflowById(req.swagger.params.identifier.value.newQuery);
```

Comment:
@lacarb I think this is a typo.  Shouldn't this be `req.swagger.params.identifier.value, newQuery` rather than `req.swagger.params.identifier.value.newQuery`?